app/src/main/java/ru/prometeydev/movie/FragmentMoviesDetails.kt

```diff
@@ -45,19 +45,19 @@ class FragmentMoviesDetails : Fragment() {
         loadData()
 
         view.findViewById<ImageView>(R.id.movie_logo)
-            .setImageResource(movie?.additional!!.pictureDrawable)
+            .load(movie?.backdrop)
```

Comment:
Why not to check movie for null once and remove all `?` later?

Reply:
Ok, I will improve